extensions/Khronos/KHR_materials_common_v2/README.md

```diff
@@ -0,0 +1,250 @@
+by the core specification. Other extensions may define additional materials.
+
+The `KHR_materials_common` extension defines three material types: Blinn-Phong,
+Lambert, and Constant.
+
+## Properties
+
+The following table lists the allowed types and ranges for the common materials:
+
+|                            |Type         | Technique(s) | Description | Required |
+|----------------------------|-------------|--------------|-------------|----------|
+|**technique**               | `BLINNPHONG`, `LAMBERT`, `CONSTANT` | — |The shading model of the material. | Yes |
+|**diffuseFactor**           | `number[4]` | `LAMBERT`, `BLINNPHONG` | The reflected diffuse RGBA factor of the material. | No, default: `[1.0,1.0,1.0,1.0]` |
+|**diffuseTexture**          | [`textureInfo`](/specification/2.0/README.md#reference-textureInfo) | `LAMBERT`, `BLINNPHONG` | The diffuse texture. | No |
+|**specularFactor**          | `number[3]` | `BLINNPHONG` | The specular RGB color of the material. | No, default: `[1.0,1.0,1.0]` |
+|**shininessFactor**         | `number`    | `BLINNPHONG` | The shininess of the material. | No, default: `1.0` |
+|**specularShininessTexture**| [`textureInfo`](/specification/2.0/README.md#reference-textureInfo) | `BLINNPHONG` | The specular shininess texture. | No |
+
+Additional properties are allowed and may lead to undefined behaviour in
+conforming viewers. Emissive and occlusion values are not defined 
+explicitly by this extension, but are inherited from the core material
+definition.
+
+## Techniques
+
+For all technique computations, the following definitions are assumed:
+*(all vectors normalized)*
+
+* `diffuseTerm` – diffuseFactor * diffuseTexture
+    * *Reflected diffuse color of the material.*
+* `specularTerm` – specularFactor * specularShininessTexture.rgb
+    * *Specular color of the material.*
+* `shineTerm` – shininessFactor * specularShininessTexture.a
+    * *Shininess component of the specular color.*
+* `emissiveTerm` – emissiveFactor * emissiveTexture
+    *  [*From core glTF material specification.*](https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#additional-maps)
+* `occlusionTerm` – occlusionFactor * occlusionTexture
+    *  [*From core glTF material specification.*](https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#additional-maps)
+* `aL` – Ambient light
+    * *Constant sum of all ambient light contributions from the scene.*
+* `N` – Normal vector
+* `L` – Light vector
+* `I` – Eye vector
+* `H` – Half-angle vector
+    * *Calculated as halfway between the unit Eye and Light vectors, using the equation `H = normalize(I+L)`.*
+
+### Constant
+
+The `CONSTANT` technique defines shading that is independent of lighting, or
+unlit. No extension properties are included with Constant materials, but
+emissive color (from the core glTF 2.0 specification) is used.
+
+The displayed color is calculated as:
+
+```
+color = emissiveTerm + occlusionTerm
```

Comment:
I think the math here is incorrect; I don't think the occlusionTerm should be added by itself.

Reply:
```
color = emissiveTerm + occlusionTerm * aL
```

Does that look correct? I'm probably not the person to be writing these equations.